components/gc-subway/aem-bad-implementation-en.html

```diff
@@ -30,7 +28,6 @@ <h1>Service name</h1>
 			<li><a href="section1-en.html">Page 6</a></li>
 		</ul>
 	</nav>
-	<h1 property="name" id="wb-cont" class="gc-thickline">Page 3</h1>
```

Comment:
I don't think this H1 should be removed. I believe it was the objective of this page to emulate a bad implementation of the Subway navigation so that we could ensure it still works after the stabilization.

Reply:
Fixed in amended commit.